demisto_sdk/commands/content_graph/interface/neo4j/queries/constraints.py

```diff
@@ -0,0 +1,50 @@
+from neo4j import Transaction
+
+from demisto_sdk.commands.content_graph.common import ContentTypes, Rel
+from demisto_sdk.commands.content_graph.interface.neo4j.queries.common import run_query
+
+
+NODE_PROPERTY_UNIQUENESS_TEMPLATE = 'CREATE CONSTRAINT IF NOT EXISTS FOR (n:{label}) REQUIRE n.{prop} IS UNIQUE'
+NODE_PROPERTY_EXISTENCE_TEMPLATE = 'CREATE CONSTRAINT IF NOT EXISTS FOR (n:{label}) REQUIRE n.{prop} IS NOT NULL'
+REL_PROPERTY_EXISTENCE_TEMPLATE = 'CREATE CONSTRAINT IF NOT EXISTS FOR ()-[r:{label}]-() REQUIRE r.{prop} IS NOT NULL'
+
+
+def create_constraints(tx: Transaction) -> None:
+    create_nodes_constraints(tx)
+    # create_relationships_constraints(tx)
```

Comment:
Needed?

Reply:
Not required, but we can create some, E.g. `HAS_COMMAND` must have the following properties: deprecated, name, description. We can make sure we are really adding them.